basics/service-ticket/settings.rst

```diff
@@ -48,6 +48,8 @@ text. (Your highlights are visible to other agents.)
 
 ----
 
+.. _ticket-submenu:
+
```

Comment:
I don't understand the meaning of this change, but when I compare the old and thew new site, I recognize a change in the distance between the header "Highlighting ticket text" and the image above. The image sticks to the header below:
![grafik](https://user-images.githubusercontent.com/97293098/213727103-df5274cc-98f0-4c1b-a2cf-c0d8402b5de9.png)

In my opinion it should look more like in the previous version:
![grafik](https://user-images.githubusercontent.com/97293098/213727253-add01a88-94ca-4775-9183-1156d387dd16.png)

Reply:
This change depends on this reference:
https://github.com/zammad/zammad-user-documentation/pull/108/files#diff-7a8f49e5e4a29da5ba6c7ec877ae13a46872a511024759413cb76b6a18759077R13

I'll have a look on the formatting part later if the rest fits.

Reply:
Yes, the rest fits.

Reply:
Just as a follow up: I can't reproduce what you're seeing.
I'll merge and check on the online version to ensure it's not a bug.

Possibly your environment is missing something or behaves differently due to OS differences.